File: server/java/gradle/src/main/java/CreatePayment.java
import com.khipu.ApiClient;
import com.khipu.ApiException;
import com.khipu.api.client.PaymentsApi;
import com.khipu.api.model.PaymentsCreateResponse;

import java.io.IOException;
import java.nio.charset.Charset;
import java.nio.file.Files;
import java.nio.file.Paths;
import java.util.HashMap;
import java.util.Map;

public class CreatePayment {
    public static void main(String[] args) throws ApiException, IOException {
        Long receiverId = Long.parseLong(readSingleLineFile("../../RECEIVER_ID").trim()); //ID de cobrador
        String secret = readSingleLineFile("../../SECRET").trim(); //llave secreta

        ApiClient apiClient = new ApiClient();
        apiClient.setKhipuCredentials(receiverId, secret);
        PaymentsApi paymentsApi = new PaymentsApi();
        paymentsApi.setApiClient(apiClient);

        Map<String, Object> options = new HashMap<>();
        options.put("transactionId", "TX-1234");
        options.put("notifyUrl", "https://micomercio.com/notify");

        PaymentsCreateResponse response = paymentsApi.paymentsPost("Pago de demo" //Motivo de la compra
                , "CLP" //Moneda
                , 100.0 //Monto
                , options
        );

        System.out.println("PAYMENT_ID: " + response.getPaymentId());
    }

    static String readSingleLineFile(String path)
            throws IOException
    {
        byte[] encoded = Files.readAllBytes(Paths.get(path));
        return new String(encoded, Charset.defaultCharset()).trim();
    }
}

File: server/php/create_payment.php
<?php
require __DIR__ . '/vendor/autoload.php';

$configuration = new Khipu\Configuration();
$configuration->setSecret(trim(file_get_contents("../SECRET")));
$configuration->setReceiverId(trim(file_get_contents("../RECEIVER_ID")));
$configuration->setPlatform('demo-client', '2.0');
# $configuration->setDebug(true);
$client = new Khipu\ApiClient($configuration);
$payments = new Khipu\Client\PaymentsApi($client);

try {
    $ops = array(
        "notify_url" => "http://mi-ecomerce.com/backend/notify",
        "transaction_id" => "TX-1234",
    );
    $response = $payments->paymentsPost('Pago de demo', 'CLP', 1000, $ops);

    print "PAYMENT_ID: " . $response->getPaymentId() . "\n";

} catch (Exception $e) {
    echo $e->getMessage();
}
